### PR TITLE
Text2dictionary

### PR DESCRIPTION
**Pull request by Gaddsky**
Text2dictionary classes added with CLI interface (only argument - path to file) and test.
StringTokenizer not used, because my solution is more general, in StringTokenizer I must specify all non-Alphabetical symbols to separate words. In strip() method they passed automatically. Deficiency of method strip() - a lot of code.

Example of usingtext in attachment
[example_text.txt](https://github.com/pasby/java-battle/files/211126/example_text.txt)

### CODE REVIEW

File: gadd/src/main/java/textDictionary/CreateDictionary.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    private Set convertToSet(List wordsArrayList) {
+        Set wordsSet = new HashSet();
+        for (int i = 0; i < wordsArrayList.size(); i++) {
+            wordsSet.add(wordsArrayList.get(i));
+        }
+        return wordsSet;
+    }
+
+    private Map countWords(Set wordsSet, List wordsList) {
+        Map result = new HashMap();
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/CreateDictionary.java]: you need to parametrize all collections

File: gadd/src/main/java/textDictionary/CreateDictionary.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        Iterator it = wordsSet.iterator();
+        while (it.hasNext()) {
+            Object checkingWord = it.next();
+            for (int i = 0; i < wordsList.size(); i++) {
+                if (checkingWord.equals(wordsList.get(i))) {
+//                    int count = result.getOrDefault(checkingWord, 0);
+                    int count;
+                    if (result.containsKey(checkingWord)) {
+                        count = (int) result.get(checkingWord) + 1;
```
Comment (pasby) [inline gadd/src/main/java/textDictionary/CreateDictionary.java:81]: after parametrizing you don't need casting at all

File: gadd/src/test/java/textDictionaryTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            check.put("a", 1);
+            check.put("into", 1);
+            check.put("tokens", 1);
+        }
+        Path path = Paths.get(System.getProperty("user.dir"), "src", "test", "java", "resources", "testfile.txt");
+        String filepath = path.toString();
+        CreateDictionary cd = new CreateDictionary(filepath);
+        Map<String, Integer> result = cd.getResult();
+        Assert.assertEquals(result, check);
+        System.out.println("textDictionaryTest passed");
```
Comment (pasby) [inline gadd/src/test/java/textDictionaryTest.java]: not needed message here

**Comment by Gaddsky** [comment]
I don't know what I done, but I done

**Pull request merged**